examples/import_git.py

```diff
@@ -28,7 +28,7 @@ def publish_data_package(git_url, count=0):
         dir_name = DIR_NAME_PREFIX + "/" + str(count)
         clone_git_repo(git_url=git_url, dir_name=dir_name)
         conf = config.read_config()
-        client.Client(data_package_path=dir_name, config=conf).publish()
+        client.Client(data_package_path=dir_name, config=conf, datavalidate=True).publish()
```

Comment:
I think the code looks slightly nicer as:

```
cl = client.Client(...)
cl.publish()
```

Also datavalidate should be an argument to publish - but that is a different issue outside of this PR.